examples/medplum-provider/src/pages/components/Encounter/EncounterStatus.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Badge } from '@mantine/core';
+import { Encounter } from '@medplum/fhirtypes';
+import classes from './EncounterStatus.module.css';
+import cx from 'clsx';
+
+interface EncounterStatusProps {
+  encounter: Encounter;
+}
+
+const getStatusClass = (status: string): string => {
+  switch (status) {
+    case 'arrived':
+    case 'in-progress':
+      return classes.green;
+    case 'finished':
+      return classes.blue;
+    default:
+      return classes.gray;
+  }
+};
+
+export const EncounterStatus = ({ encounter }: EncounterStatusProps): JSX.Element => {
```

Comment:
We have a `<StatusBadge>` component in `@medplum/react`, which would be nice to use here.  We should be able to eliminate this entire component.

The colors don't exactly line up, but I think we can change the colors in `<StatusBadge>` to align with the requirements here.

See: [`packages/react/src/StatusBadge/StatusBadge.tsx`](https://github.com/medplum/medplum/blob/main/packages/react/src/StatusBadge/StatusBadge.tsx)